eng/StyleCop.Analyzers.ruleset

```diff
@@ -136,7 +136,7 @@
     <Rule Id="SA1515" Action="None" /> <!-- Single-line comment should be preceded by blank line -->
     <Rule Id="SA1516" Action="None" /> <!-- Elements should be separated by blank line -->
     <Rule Id="SA1517" Action="Error" />
-    <Rule Id="SA1518" Action="Error" />
```

Comment:
I'm fine with this change, but is there a specific reason why we decided to change it in this PR?

Reply:
For some reason, the error started to occur for many files. Looks like the test didn't work previously and was now fixed with the update.